client/trino-cli/src/main/java/io/trino/cli/Pager.java

```diff
@@ -155,14 +199,46 @@ public static Pager create(List<String> command)
             return new Pager(process.getOutputStream(), process);
         }
         catch (IOException e) {
+            if (command.equals(LESS)) {
+                return createFallbackPager(terminal);
```

Comment:
Can we check path for command existence instead of catching `catch (IOException` broadly?

for example, we shouldn't fallback if the execve call failed at fork stage

Reply:
Can we do this in Java or would that require spawning another process? I wanted to avoid the latter.

Reply:
> we shouldn't fallback if the execve call failed at fork stage

Can you explain more? Why we shouldn't fallback? We won't be spawning external processes anymore.

Reply:
Done